src/app/file-explorer.js

```diff
@@ -20,12 +20,9 @@ var css = csjs`
     background-color  : white;
   }
   .remove             {
-    align-self        : center;
-    padding-left      : 10px;
+    float             : right;
```

Comment:
I highly recommend to **ALWAYS** use `display: flex`.
These days it is supported by all browsers http://caniuse.com/#search=flex (also click: Usage relative)

It's an absolutely superior method to do any kind of layout in the browser. It is reliable and always works.
It takes some time to pick up and a good source is this:
* https://css-tricks.com/snippets/css/a-guide-to-flexbox/
* (for a playful introduction) http://flexboxfroggy.com/

Maybe the changed styling works - if it does, sure why not merge it. I dont mind, but personally i would try to stick with flexbox. If it is supposed to fix some weird styling behavior, maybe it was introduced by [this commit](https://github.com/ethereum/remix/commit/a503dc2c412f92fb262946a548e150f199953c22) to the remix repository?